docs/changelog.rst

```diff
@@ -3,6 +3,17 @@ Changelog
 
 .. scriv-insert-here
 
+.. _changelog-2.9.0:
+
+globus-compute-sdk & globus-compute-endpoint v2.9.0
+---------------------------------------------------
+
+Bug Fixes
+^^^^^^^^^
+
+- Misconfigured endpoints can now be deleted with the ``--force`` flag
```

Comment:
"Some misconfigured endpoints can now be deleted with the ``--force`` flag without encountering errors"?

So users won't confusingly read this as a new --force flag or that all misconfigured endpoints were not deletable previously.

Reply:
Perfect